Dockerfile

```diff
@@ -1,14 +1,23 @@
 ARG repository=rapidsai-dev-nightly
-ARG version=0.11-cuda10.0-devel-ubuntu18.04-py3.7
+ARG version=0.12-cuda10.0-devel-ubuntu18.04-py3.7
 
 FROM rapidsai/${repository}:${version}
 
-ADD . /clx/
+ADD . /rapids/clx/
 
 SHELL ["/bin/bash", "-c"]
+
 RUN source activate rapids \
-    && cd /clx \
+    && conda install panel=0.6.* geopandas pyppeteer \
+    && cd /rapids \
+    && git clone https://github.com/rapidsai/cuxfilter.git /rapids/cuxfilter \
+    && git clone https://github.com/rapidsai/cudatashader.git  /rapids/cudatashader \
+    && cd /rapids/cuxfilter/python \
+    && pip install -e . \
+    && cd /rapids/cudatashader \
+    && pip install -e . \
+    && cd /rapids/clx \
     && python setup.py install
 
-WORKDIR /clx
-CMD source activate rapids && sh /rapids/notebooks/utils/start-jupyter.sh
\ No newline at end of file
+WORKDIR /rapids/clx
```

Comment:
Does the limit everyone to only operate in this directory (via Juypter Notebook)?

Reply:
yes it does. i'll update it to `/rapids`. thx.